lab-kyle/test/storage-test.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
```

Comment:
These look great so far, you'll want to write a test for the delete functionality as well.